config/routes.rb

```diff
@@ -7,6 +7,9 @@
 
   namespace :account do
     get '/dashboard', to: 'dashboard#index'
+    get '/users', to: 'users#edit'
```

Comment:
resource pls

Reply:
must be resource :profile, only: [:edit, :update]  ....